Misconfigured paths since 1.1.0

Hi everybody,

I currently installed version 1.1.0 of your package and ran into this error message:

```
Error: Cannot find module '/home/xyz/node_modules/mailpit-api/dist/mjs/index.js' imported from ...
````

When I downgrade to 1.0.6, the files are in the correct place.

| 1.0.6 | 1.1.0 |
| --- | --- |
| ![Image](https://github.com/user-attachments/assets/a93bc382-5b32-4935-a3d1-8070131daa04) | ![Image](https://github.com/user-attachments/assets/a6d1e49a-395b-4906-a0e8-bf62d1054d02) |

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "mailpit-api",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "description": "A NodeJS client library, written in TypeScript, to interact with the Mailpit API.",
   "repository": {
     "type": "git",
@@ -18,7 +18,7 @@
     "test": "echo \"TODO: Add tests\" && exit 0",
     "build": "rm -fr dist/* && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && ./fixup_type",
     "pretty": "npx prettier . --write",
-    "lint": "npx eslint . --fix"
+    "lint": "npx eslint --fix src"
   },
   "keywords": [
     "mailpit-api",
@@ -43,7 +43,7 @@
     "eslint": "^9.20.1",
     "jest": "^29.7.0",
     "prettier": "3.4.2",
-    "tsx": "^4.19.2",
+    "tsx": "^4.19.3",
     "typescript": "^5.7.3",
     "typescript-eslint": "^8.24.0"
   }

--- a/tsconfig-base.json
+++ b/tsconfig-base.json
@@ -13,13 +13,13 @@
     "noFallthroughCasesInSwitch": true,
     "pretty": true,
     "resolveJsonModule": true,
-    "rootDir": ".",
+    "rootDir": "src",
     "skipLibCheck": true,
     "strict": true,
     "traceResolution": false,
     "types": ["node", "jest"]
   },
   "compileOnSave": false,
-  "include": ["src", "eslint.config.mjs", "test"],
-  "exclude": ["node_modules", "dist"]
+  "exclude": ["node_modules", "dist", "test"],
+  "include": ["src"]
 }

